### PR TITLE
[tools]: add more functional extraction tool for camera.

### PR DESCRIPTION
**Pull request by gchen-apollo**

### CODE REVIEW

File: modules/tools/sensor_calibration/extract_data.py [hunk 1]
```diff
@@ -352,32 +355,32 @@ def parse_channel_config(channels):
...
-        odometry_subfolder = [x for x in subfolders if'_odometry' in x]
+                    if '_apollo_sensor_' in x]
+        odometry_subfolders = [x for x in subfolders if'_odometry' in x]
+        lidar_subfolders = [x for x in subfolders if '_PointCloud2' in x]
         print(lidar_subfolders)
-        print(odometry_subfolder)
+        print(odometry_subfolders)
         if len(lidar_subfolders) is 0 or \
-            len(odometry_subfolder) is not 1:
+            len(odometry_subfolders) is not 1:
```
Comment (xiaoxq) [inline modules/tools/sensor_calibration/extract_data.py]: Fit in one line.

File: modules/tools/sensor_calibration/extract_static_data.py [hunk 2]
```diff
@@ -266,52 +272,20 @@ def select_static_image_pcd(path, min_distance=5, stop_times=5,
...
         lidar_idx = [ camera_lidar_nearest_pairs[x] for x in image_idx ]
-        output_path = os.path.join(camera_folder_path, 'camera-lidar-pairs')
+        output_path = os.path.join(camera_folder_path, output_folder_name)
         mkdir_p(output_path)
         for count, i in enumerate(image_idx):
             #  save images
-            in_file = os.path.join(camera_folder_path, str(int(i)).zfill(6)+image_suffix)
-            out_file = os.path.join(output_path, str(int(count)).zfill(6)+image_suffix)
+            in_file = os.path.join(camera_folder_path, str(int(i)).zfill(Z_FILL_LEN)+image_suffix)
+            out_file = os.path.join(output_path, str(int(count)).zfill(Z_FILL_LEN)+image_suffix)
```
Comment (xiaoxq) [inline modules/tools/sensor_calibration/extract_static_data.py]: Spaces around `+`

File: modules/tools/sensor_calibration/extract_static_data.py [hunk 3]
```diff
@@ -266,52 +272,20 @@ def select_static_image_pcd(path, min_distance=5, stop_times=5,
...
             copyfile(in_file, out_file)
             j = camera_lidar_nearest_pairs[i]
             #  save pcd
-            in_file = os.path.join(path, lidar_subfolder, str(int(j)).zfill(6)+pcd_suffix)
-            out_file = os.path.join(output_path, str(int(count)).zfill(6)+pcd_suffix)
+            in_file = os.path.join(path, lidar_subfolder, str(int(j)).zfill(Z_FILL_LEN)+pcd_suffix)
+            out_file = os.path.join(output_path, str(int(count)).zfill(Z_FILL_LEN)+pcd_suffix)
```
Comment (xiaoxq) [inline modules/tools/sensor_calibration/extract_static_data.py]: ditto

File: modules/tools/sensor_calibration/config/lidar_to_gnss_sample_config.yaml [hunk 4]
```diff
@@ -10,11 +10,12 @@ odometry_file: "./_sensor_novatel_Odometry/odometry"
...
 #initial calibration paramters, including tranlation and raotation, to project a point in source
 #source_sensor coordinate to destination_sensor coordinate. translation is in meter. rotation is in quaternion
-translation:
+transform:
+  translation:
        x: 0.0
        y: 0.0
        z: 0.0
-rotation:
+  rotation:
```
[the remark this answers is not in the document]
Comment (gchen-apollo) [reply modules/tools/sensor_calibration/config/lidar_to_gnss_sample_config.yaml]: done

**Pull request merged**